official/recommendation/ncf_test.py

```diff
@@ -185,34 +185,38 @@ def test_hit_rate_and_ndcg(self):
     self.assertAlmostEqual(ndcg, (1 + math.log(2) / math.log(3) +
                                   2 * math.log(2) / math.log(4)) / 4)
 
-  _BASE_END_TO_END_FLAGS = ['-batch_size', '1024', '-train_epochs', '1']
+  _BASE_END_TO_END_FLAGS_ESTIMATOR = [
```

Comment:
Once we have the rounding behavior in Keras, we won't need to have separate flags for Keras and Estimator.

Reply:
Done.